component-library/component-lib/src/lib/form-components/multi-checkbox/multi-checkbox.component.ts

```diff
@@ -34,123 +38,223 @@ export class MultiCheckboxComponent implements OnInit {
 
   @Input() config: IMultiCheckboxConfig = {
```

Comment:
Need to have multiple ways to input values here; follow the same pattern as all other input components (config and @Input() for each field in the config with an override in the constructor or onInit() that sets the values in the config to the values given in the individual @Inputs(), if they exist.